src/main/java/edu/uc/inventorymanager/dto/Item.java

```diff
@@ -1,12 +1,15 @@
 package edu.uc.inventorymanager.dto;
 
 import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor
 public @Data class Item {
-    private int id;
+    @NonNull private int id;
```

Comment:
@NonNull tells @RequiredArgsConstructor that this field is one of the required arguments